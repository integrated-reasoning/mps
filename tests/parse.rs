mod tests {
  use color_eyre::Result;
  use num_traits::float::Float;
  cfg_if::cfg_if! {
    if #[cfg(feature = "trace")] {
      use nom_locate::LocatedSpan;
      use nom_tracable::TracableInfo;
    }
  }

  fn parse<T: Float>(input: &'static str) -> Result<mps::Parser<'_, f32>> {
    cfg_if::cfg_if! {
      if #[cfg(feature = "trace")] {
        let info = TracableInfo::new().forward(false).backward(false);
        let (_, parsed) = mps::Parser::<T>::parse(LocatedSpan::new_extra(input, info))?;
      } else {
        let (_, parsed) = mps::Parser::<T>::parse(&input)?;
      }
    }
    Ok(parsed)
  }

  #[test]
  fn test_parse_agg() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/agg"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_ship04l() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/ship04l"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_d2q06c() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/d2q06c"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_e226() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/e226"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_nl25fv47() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/25fv47"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_bore3d() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/bore3d"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_ganges() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/ganges"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_adlittle() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/adlittle"
    ))?);
    Ok(())
  }

  #[ignore] // TODO: Fix (fails in row_line and columns)
  fn _test_parse_forplan() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/forplan"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_sc205() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/sc205"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_nl80bau3b() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/80bau3b"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_scrs8() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/scrs8"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_wood1p() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/wood1p"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_boeing1() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/boeing1"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_kb2() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/kb2"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_ship08s() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/ship08s"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_scfxm1() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/scfxm1"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_agg2() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/agg2"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_finnis() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/finnis"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_dfl001() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/dfl001"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_pilot87() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/pilot87"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_sctap1() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/sctap1"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_agg3() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/agg3"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_grow7() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/grow7"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_scorpion() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/scorpion"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_maros() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/maros"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_shell() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/shell"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_greenbeb() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/greenbeb"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_sc50b() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/sc50b"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_recipe() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/recipe"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_sierra() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/sierra"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_scagr25() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/scagr25"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_modszk1() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/modszk1"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_ship12l() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/ship12l"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_stair() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/stair"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_cycle() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/cycle"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_sc105() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/sc105"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_pilot_ja() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/pilot.ja"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_beaconfd() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/beaconfd"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_czprob() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/czprob"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_pilot_we() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/pilot.we"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_standgub() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/standgub"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_standmps() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/standmps"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_scsd8() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/scsd8"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_woodw() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/woodw"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_scsd6() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/scsd6"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_scsd1() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/scsd1"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_share2b() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/share2b"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_gfrd_pnc() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/gfrd-pnc"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_bnl2() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/bnl2"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_stocfor2() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/stocfor2"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_nesm() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/nesm"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_share1b() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/share1b"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_ship04s() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/ship04s"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_grow15() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/grow15"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_maros_r7() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/maros-r7"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_blend() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/blend"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_lotfi() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/lotfi"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_standata() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/standata"
    ))?);
    Ok(())
  }

  ////#[test]
  ////fn test_parse_d6cube() -> Result<()> {
  //  insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
  //    "../tests/data/netlib/d6cube"
  //  ))?);
  //  Ok(())
  ////}

  #[test]
  fn test_parse_degen3() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/degen3"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_capri() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/capri"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_grow22() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/grow22"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_etamacro() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/etamacro"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_ship08l() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/ship08l"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_afiro() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/afiro"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_degen2() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/degen2"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_boeing2() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/boeing2"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_fit1d() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/fit1d"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_scfxm2() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/scfxm2"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_sctap3() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/sctap3"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_fit1p() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/fit1p"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_pilot() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/pilot"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_fit2d() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/fit2d"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_sctap2() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/sctap2"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_scfxm3() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/scfxm3"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_brandy() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/brandy"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_greenbea() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/greenbea"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_tuff() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/tuff"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_sc50a() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/sc50a"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_vtp_base() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/vtp.base"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_pilotnov() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/pilotnov"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_ship12s() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/ship12s"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_seba() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/seba"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_fffff800() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/fffff800"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_israel() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/israel"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_perold() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/perold"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_pilot4() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/pilot4"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_scagr7() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/scagr7"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_bandm() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/bandm"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_bnl1() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/bnl1"
    ))?);
    Ok(())
  }

  #[test]
  fn test_parse_stocfor1() -> Result<()> {
    insta::assert_yaml_snapshot!(parse::<f32>(include_str!(
      "../tests/data/netlib/stocfor1"
    ))?);
    Ok(())
  }
}
