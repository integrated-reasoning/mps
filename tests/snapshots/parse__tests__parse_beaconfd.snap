---
source: tests/parse.rs
expression: "parse::<f32>(include_str!(\"../tests/data/netlib/beaconfd\"))?"
---
name: BEACONFD
rows:
  - row_type: Nr
    row_name: 11CSTR
  - row_type: Leq
    row_name: "50022"
  - row_type: Leq
    row_name: "50024"
  - row_type: Leq
    row_name: "50028"
  - row_type: Leq
    row_name: "50055"
  - row_type: Leq
    row_name: "50059"
  - row_type: Leq
    row_name: "50077"
  - row_type: Leq
    row_name: "50080"
  - row_type: Leq
    row_name: "50081"
  - row_type: Leq
    row_name: "50109"
  - row_type: Leq
    row_name: "50110"
  - row_type: Leq
    row_name: "50139"
  - row_type: Leq
    row_name: "50157"
  - row_type: Leq
    row_name: "50165"
  - row_type: Leq
    row_name: "50168"
  - row_type: Leq
    row_name: "50178"
  - row_type: Leq
    row_name: "50190"
  - row_type: Leq
    row_name: "50192"
  - row_type: Leq
    row_name: "50195"
  - row_type: Leq
    row_name: "50302"
  - row_type: Leq
    row_name: "50303"
  - row_type: Leq
    row_name: "50325"
  - row_type: Leq
    row_name: "50412"
  - row_type: Leq
    row_name: "50448"
  - row_type: Leq
    row_name: "50470"
  - row_type: Leq
    row_name: "50477"
  - row_type: Leq
    row_name: "50495"
  - row_type: Leq
    row_name: "50545"
  - row_type: Leq
    row_name: "50548"
  - row_type: Leq
    row_name: "50623"
  - row_type: Leq
    row_name: "50842"
  - row_type: Leq
    row_name: "50854"
  - row_type: Leq
    row_name: "50861"
  - row_type: Leq
    row_name: "50917"
  - row_type: Eq
    row_name: "51022"
  - row_type: Eq
    row_name: "51024"
  - row_type: Eq
    row_name: "51025"
  - row_type: Eq
    row_name: "51026"
  - row_type: Eq
    row_name: "51027"
  - row_type: Eq
    row_name: "51028"
  - row_type: Eq
    row_name: "51033"
  - row_type: Eq
    row_name: "51041"
  - row_type: Eq
    row_name: "51043"
  - row_type: Eq
    row_name: "51055"
  - row_type: Eq
    row_name: "51059"
  - row_type: Eq
    row_name: "51077"
  - row_type: Eq
    row_name: "51080"
  - row_type: Eq
    row_name: "51081"
  - row_type: Eq
    row_name: "51109"
  - row_type: Eq
    row_name: "51110"
  - row_type: Eq
    row_name: "51125"
  - row_type: Eq
    row_name: "51139"
  - row_type: Eq
    row_name: "51144"
  - row_type: Eq
    row_name: "51150"
  - row_type: Eq
    row_name: "51157"
  - row_type: Eq
    row_name: "51165"
  - row_type: Eq
    row_name: "51168"
  - row_type: Eq
    row_name: "51176"
  - row_type: Eq
    row_name: "51178"
  - row_type: Eq
    row_name: "51190"
  - row_type: Eq
    row_name: "51191"
  - row_type: Eq
    row_name: "51192"
  - row_type: Eq
    row_name: "51195"
  - row_type: Eq
    row_name: "51302"
  - row_type: Eq
    row_name: "51303"
  - row_type: Eq
    row_name: "51325"
  - row_type: Eq
    row_name: "51340"
  - row_type: Eq
    row_name: "51400"
  - row_type: Eq
    row_name: "51412"
  - row_type: Eq
    row_name: "51427"
  - row_type: Eq
    row_name: "51446"
  - row_type: Eq
    row_name: "51448"
  - row_type: Eq
    row_name: "51460"
  - row_type: Eq
    row_name: "51470"
  - row_type: Eq
    row_name: "51477"
  - row_type: Eq
    row_name: "51490"
  - row_type: Eq
    row_name: "51495"
  - row_type: Eq
    row_name: "51496"
  - row_type: Eq
    row_name: "51545"
  - row_type: Eq
    row_name: "51548"
  - row_type: Eq
    row_name: "51605"
  - row_type: Eq
    row_name: "51620"
  - row_type: Eq
    row_name: "51623"
  - row_type: Eq
    row_name: "51625"
  - row_type: Eq
    row_name: "51635"
  - row_type: Eq
    row_name: "51812"
  - row_type: Eq
    row_name: "51824"
  - row_type: Eq
    row_name: "51827"
  - row_type: Eq
    row_name: "51835"
  - row_type: Eq
    row_name: "51842"
  - row_type: Eq
    row_name: "51854"
  - row_type: Eq
    row_name: "51857"
  - row_type: Eq
    row_name: "51861"
  - row_type: Eq
    row_name: "51884"
  - row_type: Eq
    row_name: "51917"
  - row_type: Eq
    row_name: "51919"
  - row_type: Eq
    row_name: "51922"
  - row_type: Eq
    row_name: "51940"
  - row_type: Eq
    row_name: "51943"
  - row_type: Eq
    row_name: "51945"
  - row_type: Eq
    row_name: "51947"
  - row_type: Eq
    row_name: "51974"
  - row_type: Eq
    row_name: "51981"
  - row_type: Eq
    row_name: "51990"
  - row_type: Eq
    row_name: "60080"
  - row_type: Eq
    row_name: "60157"
  - row_type: Eq
    row_name: "60545"
  - row_type: Eq
    row_name: "60854"
  - row_type: Eq
    row_name: "609010"
  - row_type: Eq
    row_name: "609012"
  - row_type: Eq
    row_name: "609022"
  - row_type: Eq
    row_name: "609032"
  - row_type: Eq
    row_name: "609101"
  - row_type: Eq
    row_name: "609111"
  - row_type: Eq
    row_name: "609138"
  - row_type: Eq
    row_name: "609155"
  - row_type: Eq
    row_name: "609156"
  - row_type: Eq
    row_name: "609158"
  - row_type: Eq
    row_name: "609161"
  - row_type: Eq
    row_name: "609166"
  - row_type: Eq
    row_name: "609172"
  - row_type: Eq
    row_name: "609173"
  - row_type: Eq
    row_name: "609202"
  - row_type: Eq
    row_name: "609221"
  - row_type: Eq
    row_name: "609242"
  - row_type: Eq
    row_name: "609246"
  - row_type: Eq
    row_name: "609252"
  - row_type: Eq
    row_name: "609280"
  - row_type: Eq
    row_name: "609330"
  - row_type: Eq
    row_name: "609340"
  - row_type: Eq
    row_name: "609361"
  - row_type: Eq
    row_name: "609362"
  - row_type: Eq
    row_name: "609363"
  - row_type: Eq
    row_name: "609371"
  - row_type: Eq
    row_name: "609380"
  - row_type: Eq
    row_name: "609402"
  - row_type: Eq
    row_name: "609422"
  - row_type: Eq
    row_name: "609441"
  - row_type: Eq
    row_name: "609451"
  - row_type: Eq
    row_name: "609463"
  - row_type: Eq
    row_name: "609475"
  - row_type: Eq
    row_name: "609490"
  - row_type: Eq
    row_name: "609494"
  - row_type: Eq
    row_name: "609496"
  - row_type: Eq
    row_name: "609497"
  - row_type: Eq
    row_name: "609540"
  - row_type: Eq
    row_name: "609563"
  - row_type: Eq
    row_name: "609601"
  - row_type: Eq
    row_name: "609602"
  - row_type: Eq
    row_name: "609608"
  - row_type: Eq
    row_name: "609609"
  - row_type: Eq
    row_name: "609610"
  - row_type: Eq
    row_name: "609615"
  - row_type: Eq
    row_name: "609625"
  - row_type: Eq
    row_name: "609630"
  - row_type: Eq
    row_name: "609632"
  - row_type: Eq
    row_name: "609633"
  - row_type: Eq
    row_name: "609634"
  - row_type: Eq
    row_name: "609636"
  - row_type: Eq
    row_name: "609650"
  - row_type: Eq
    row_name: "609652"
  - row_type: Eq
    row_name: "609653"
  - row_type: Eq
    row_name: "609654"
  - row_type: Eq
    row_name: "609655"
  - row_type: Eq
    row_name: "609712"
  - row_type: Eq
    row_name: "609715"
  - row_type: Eq
    row_name: "609721"
  - row_type: Eq
    row_name: "609723"
  - row_type: Eq
    row_name: "609725"
  - row_type: Eq
    row_name: "609727"
  - row_type: Eq
    row_name: "609729"
  - row_type: Eq
    row_name: "609731"
  - row_type: Eq
    row_name: "609750"
  - row_type: Eq
    row_name: "609771"
columns:
  - name: "10022"
    first_pair:
      row_name: "50022"
      value: 0.05
    second_pair:
      row_name: "51022"
      value: -1
rhs: ~
ranges: ~
bounds: ~

